config.json

```diff
@@ -34,7 +34,7 @@
       "description": "Specifies the distro rid for Unix OS.",
       "valueType": "property",
       "values": [],
-      "defaultValue": "${OSRid}"
+      "defaultValue": "${OSRid}-${CPUArch}"
```

Comment:
The changes here mean that /p:DistroRid will be set to something like 'osx.10-11-x64' OSX (previously we were passing an empty value), but it looks like that will be OK?  @eerhardt or @gkhanna79 do you have a sense here?